README.md

```diff
@@ -41,17 +41,14 @@ Test Matrix
 ------------
 | Andrid Studio Version | cmake in SDK| NDK      | Result  |
```

Comment:
I would get rid of this table. We already say that users should use AS 3.0+, and the link I provide in the comment above describes how to install CMake.

Reply:
for open source project, like to get to what we tested with. Mainly there are so many combinations and users would always have different combinations etc., when problem happens, we would know at the time what was tested with, so we could go that configuration to make sure it is still working, then debugging the new combinations